### DISCUSSION TITLE
Question about fr24feed

### DISCUSSION THREAD

**Discussion by garethhowell** [ketilmo/balena-ads-b#64] [Q&A] [answered]
Now that flightradar has requested us not to feed mlat data, I see that the defaults in fr24feed.ini are now to have `mlat="no"`.

I have an installed node that I need to change this setting. Unfortunately I can't see how in a sensible fashion as there is no editor installed in the container. I could simple `echo` over it, but this is error prone. Is there a better way?

**Comment by ketilmo** [comment, marked as answer]
Hi @garethhowell,

Thanks for reaching out. If you have installed balena-ads-b without modifying anything in the source code, the most straightforward approach would be to update to the latest version. All your settings stored in environment variables will be retained. See the following instructions for details about updating: https://github.com/ketilmo/balena-ads-b#part-11--updating-to-the-latest-version

Let us know if this works for you.

Kind regards,
Ketil

**Reply by garethhowell** [reply]
Cheers, mate.
I asked after I had set off an update. I thought it had completed, but looking at it now, I see it is still in progress.

Gareth
